packages/amplify-provider-awscloudformation/src/admin-modelgen.ts

```diff
@@ -43,14 +43,17 @@ export const adminModelgen = async (context: $TSContext, resources: $TSAny[]): P
     },
   };
   const originalStdoutWrite = process.stdout.write;
+  let tempStdoutWrite: fs.WriteStream = null;
   try {
     // overwrite project config with config that forces codegen to output js to a temp location
     stateManager.setProjectConfig(undefined, forceJSCodegenProjectConfig);
 
     // generateModels and generateModelIntrospection print confusing and duplicate output when executing these codegen paths
     // so pipe stdout to a file and then reset it at the end to suppress this output
     await fs.ensureDir(absoluteTempOutputDir);
-    const tempStdoutWrite = fs.createWriteStream(path.join(absoluteTempOutputDir, 'temp-console-log.txt'));
+    const tempLogFilePath = path.join(absoluteTempOutputDir, 'temp-console-log.txt');
+    await fs.ensureFile(tempLogFilePath);
```

Comment:
createWriteStream's lazy file creation didn't work well on windows in this particular case.